libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, Story } from '@storybook/react';
+import { AnnouncementBanner } from '.';
+
+export default {
+  component: AnnouncementBanner,
+  title: 'Components/AnnouncementBanner',
+} as Meta;
+
+const Template: Story = (args) => {
+  return (
+    <AnnouncementBanner {...args}>
+      <div className="text-center text-xs font-semibold md:text-sm">
+        Permissionless deployment is coming to Flow! Read more{' '}
+        <span style={{ textDecoration: 'underline' }}>
+          <a href="https://permissionless.onflow.org/">here</a>
```

Comment:
Should this have a target="_blank" to open a new window? Assuming this component will be added to the site later?

Reply:
Yeah that would a good idea, nice catch!